apiClientDotNetTest/Resources/testConfigPsdevTwo.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "sessionAuthHost": "psdev.symphony.com",
+  "sessionAuthPort": 443,
+  "keyAuthHost": "psdev.symphony.com",
+  "keyAuthPort": 443,
+  "podHost": "psdev.symphony.com",
+  "podPort": 443,
+  "agentHost": "psdev.symphony.com",
+  "agentPort": 443,
+  "appPrivateKeyPath": "C:/TEST/gitlab/certs/",
+  "appPrivateKeyName": "privatekey.pkcs8",
+  "appId": "gitlab-eis",
+  "botPrivateKeyPath": "C:/TEST/gitlab/certs/",
```

Comment:
If this file is not provided (which it should not), it should not be referred to, and an explanation should be given to the user, so that he can generate/update the file

Reply:
I agree and I will then create an integration config template file.